logos/src/logos/main.py

```
@@ -3624,24 +3735,146 @@ async def add_model(data: GetPolicyRequest):
     with DBManager() as db:
         return db.get_policy(**data.dict()), 200
 
+@app.post("/logosdb/get_api_key_id", tags=["admin"])
+async def get_api_key_id(data: GetApiKeyIdRequest):
```

Flake8 (7.3.0)

[error] 'GetApiKeyIdRequest' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)